moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -959,6 +968,62 @@ void ompl_interface::ModelBasedPlanningContext::unregisterTerminationCondition()
   ptc_ = nullptr;
 }
 
+void ompl_interface::ModelBasedPlanningContext::omplPlannerStatusToMoveItErrorCode(
+    og::SimpleSetupPtr ompl_simple_setup, moveit_msgs::msg::MoveItErrorCodes& res)
+{
+  ompl::base::PlannerStatus ompl_status = ompl_simple_setup->getLastPlannerStatus();
+  if (ompl_status == ompl::base::PlannerStatus::UNKNOWN)
+  {
+    RCLCPP_WARN(LOGGER, "Motion planning failed for an unknown reason");
+    res.val = moveit_msgs::msg::MoveItErrorCodes::PLANNING_FAILED;
+  }
+  else if (ompl_status == ompl::base::PlannerStatus::INVALID_START)
+  {
+    RCLCPP_WARN(LOGGER, "Invalid start state");
+    res.val = moveit_msgs::msg::MoveItErrorCodes::PLANNING_FAILED;
+  }
+  else if (ompl_status == ompl::base::PlannerStatus::INVALID_GOAL)
+  {
+    RCLCPP_WARN(LOGGER, "Invalid goal state");
+    res.val = moveit_msgs::msg::MoveItErrorCodes::PLANNING_FAILED;
```

Comment:
Same here. Would GOAL_CONSTRAINTS_VIOLATED work?

Reply:
I feel like that's slightly different. Prob best to add a new code that's an exact match